sorc/fvcom_tools.fd/module_nwp_base.f90

```diff
@@ -105,7 +105,7 @@ subroutine alloc_obsbase(this,numvar,ifquality)
             if(this%ifquality) allocate(this%quality(numvar))
 
          else
-            write(*,*) 'alloc_obsbase Error: dimension must be larger than 0:', numvar
```

Comment:
It looks like the only change in this file is "write(*,*) -> write(6,*)" for many lines.  Is the output not shown correctly with write(*,*)?

Reply:
write(6,*) will direct the information to different stdout files for each core. Write(*,*) will write all information to one stdout file. write(*,*) is the same as write(6.*) in single core job but it will make stdout file very messy when multiple cores are used.